core/tests/protractor/collections.js

```diff
@@ -30,6 +30,10 @@ describe('Collections', function() {
       '.protractor-test-reload-collection-button')).first().click();
     general.acceptAlert();
     browser.waitForAngular();
+    element.all(by.css(
```

Comment:
Do you think we could push some of these to admin.js -- e.g. admin.reloadAllExplorations(), admin.reloadCollection(index)?